sdks/java/io/jms/src/test/java/org/apache/beam/sdk/io/jms/JmsIOTest.java

```diff
@@ -58,13 +63,26 @@
   @Rule
   public final transient TestPipeline pipeline = TestPipeline.create();
 
+  @Rule
+  public ExpectedException expectedException = ExpectedException.none();
+
   @Before
   public void startBroker() throws Exception {
     broker = new BrokerService();
     broker.setUseJmx(false);
     broker.setPersistenceAdapter(new MemoryPersistenceAdapter());
     broker.addConnector(BROKER_URL);
     broker.setBrokerName("localhost");
+    broker.setPopulateJMSXUserID(true);
+    broker.setUseAuthenticatedPrincipalForJMSXUserID(true);
+
+    // enable authentication
+    List<AuthenticationUser> users = new ArrayList<>();
+    users.add(new AuthenticationUser("test", "test", "users,admins"));
```

Comment:
can you document what parameters these are? e.g.,

`/* user */ test, /* password */ test, /* groups */ users, admins`

of course with the right parameter names and quotes, etc.